openmetadata-docs/content/connectors/database/datalake/airflow.md

```diff
@@ -193,6 +199,49 @@ The `sourceConfig` is defined [here](https://github.com/open-metadata/OpenMetada
 - `includeViews`: true or false, to ingest views definitions.
 - `databaseFilterPattern`, `schemaFilterPattern`, `tableFilternPattern`: Note that the they support regex as include or exclude. E.g.,
 
+This is a sample config for Datalake using Azure:
+
+```yaml
+# Datalake with Azure 
+
+source:
+  type: datalake
+  serviceName: local_datalake
+  serviceConnection:
+    config:
+      type: Datalake
+      configSource:      
+        securityConfig: 
+          clientId: client-id
+          clientSecret: client-secret
+          tenantId: tenant-id
+          accountName: account-name
+      prefix: prefix
+  sourceConfig:
+    config:
+      tableFilterPattern:
+        includes:
+        - ''
+sink:
+  type: metadata-rest
+  config: {}
+workflowConfig:
+  openMetadataServerConfig:
+    hostPort: <OpenMetadata host and port>
+    authProvider: <OpenMetadata auth provider>
+```
+
+#### Source Configuration - Service Connection using Azure
+
+The `sourceConfig` is defined [here](https://github.com/open-metadata/OpenMetadata/blob/main/openmetadata-spec/src/main/resources/json/schema/security/credentials/azureCredentials.json).
+
+- **Client ID** : Client ID of the data storage account
+- **Client Secret** : Client Secret of the account
+- **Tenant ID** : Tenant ID under which the data storage account falls
+- **Account Name** : Account Name of the data Storage
+
+**schemaFilterPattern** and **tableFilternPattern**: Note that the `schemaFilterPattern` and `tableFilterPattern` both support regex as `include` or `exclude`. E.g.,
```

Comment:
let's add some more details regarding filter patterns here that we are using schema filter pattern to filter out the buckets and table filter pattern is used to filter files

Reply:
This is missing for gcs and s3 as well so let's add it there as well

Reply:
Ok . Let me do that.